common/source/docs/common-cameras-and-gimbals.rst

```diff
@@ -17,45 +17,56 @@ Gimbals
 
 ArduPilot supports both brushless direct drive gimbals (Tarot, SimpleBGC, SToRM32)
 that have their own self-stabilization controllers and the simpler servo-driven
-gimbals in which ArduPilot controls the stabilisation.
+gimbals in which ArduPilot controls the stabilisation. Some gimbals also integrate still or video cameras.
 
+-  :ref:`Servo Gimbals <common-camera-gimbal>` — older-style servo-driven gimbal where ArduPilot provides stabilisation
 -  :ref:`Brushless PWM <common-brushless-pwm-gimbal>` - brushless gimbals that accept PWM or SBUS input for angle control
+
```

Comment:
I'd prefer we don't have this separation.  It disturbs the flow and I don't think that users will really care about the distinction between the more DIY (above) and the OEM below.

Reply:
I think that recombining the Gimbals is okay...and will do